.github/workflows/azure-pipelines.yml

```diff
@@ -0,0 +1,71 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+
+trigger:
+- main
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  solution: '**/*.csproj'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+  agentVersion: '2.6.4.2' #Agent version that is going to be used
+  projectDirectory: '.' #path to one project in your solution
+  webAppName: 'insecure-bank' #your app service name
+  consoleURL: 'http://console-host:8089/hdiv-console-services' #Console address. Set to empty if you do not want to use the console
+  consoleToken: 'token-value' #console token provided by Hdiv
+  enableToolbar: 'true' #Set value to 'true' to enable the toolbar 
+  serverName: 'serverName' #Select a name for your server
+  isLinux: 'false' #Set to true if you app service runs under linux
```

Comment:
Can't platform be detected?

Reply:
That would be cool. However, I don't think that this would possible or, at least, I have not found a way to do it.

Reply:
Another option would be to use two yaml files for each environment